src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/LoggingExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.UnitTestApp
+{
+    internal static class LoggingExtensions
+    {
+        private static readonly Action<ILogger, Exception> _scenarioReady =
```

Comment:
While I understand this is objectively better, is it necessary for our tests? It's a lot simpler to just LogInformation.

Reply:
I simplified this a bit. We're using the EventId in the test host to be able to identify the different event types, so it's better to have those IDs consolidated into one file.